platform/viewer/public/config/local_static.js

```diff
@@ -12,9 +13,9 @@ window.config = {
       sourceName: 'dicomweb',
       configuration: {
         name: 'DCM4CHEE',
-        wadoUriRoot: '/dicomweb',
-        qidoRoot: '/dicomweb',
-        wadoRoot: '/dicomweb',
+        wadoUriRoot: 'http://localhost:5000/dicomweb',
```

Comment:
Having the yarn start serve up the dicomweb file results in a number of issues, so I'm separating it out.  This is only for testing right now.